eo-runtime/pom.xml

```diff
@@ -118,6 +118,7 @@ SOFTWARE.
               <goal>register</goal>
               <goal>assemble</goal>
               <goal>transpile</goal>
+              <goal>latex</goal>
```

Comment:
@potatmen We don't need to add new goal to our runtime process. It will be a standalone Mojo